### PR TITLE
test: Add tests for private methods and accessors productions

### PR DESCRIPTION
**Pull request by mkubilayk**
Adds coverage for semantics of private methods and private accessors according to the test plan posted in #1343.

> - Semantics > Private method constructs
>   - Private ordinary methods
>   - Private accessors

I used private fields as an intermediate storage to test the behaviour of private methods. Let me know if that makes the tests too tightly coupled. 

---
@leobalter @rwaldron for review.
cc: @littledan @jbhoosreddy

### CODE REVIEW

**Comment by littledan** [comment]
IMO it's not too tightly coupled, as the private methods proposal builds on the private fields proposal (explicitly, the specification says this).

File: src/class-elements/rs-private-getter-alt.case [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+get #ZW_‌_NJ() {
+  return this.#ZW_‌_NJ_;
+}
+get #ZW_‍_J() {
+  return this.#ZW_‍_J_;
+}
+
+//- privateinspectionfunctions
+$(value) {
+  this.#$_ = value;
```
Comment (rwaldron) [inline src/class-elements/rs-private-getter-alt.case]: No such private field exists. You'll need to this: 

```
#$_; #__; #\u{6F}_; #℘_; #ZW_‌_NJ_; #ZW_‍_J_;
```
...under the `//- fields` label.

Or you can just `return 1;` from all of the getters and remove these assignments. 

Note that this applies to all of the case files in this changeset. I recommend running `./make.py build` to verify the resulting test material is valid.

**Comment by mkubilayk** [comment]
Added missing field declarations.

**Comment by mkubilayk** [comment]
@leobalter: Sure, no problem! Thanks for taking the time to review this. I pushed the generated tests.

**Pull request merged**